src/Dockerfile

```diff
@@ -0,0 +1,4 @@
+# NOTE: golang > 1.14 drops darwin 32-bit support
```

Comment:
Is 32-bit Darwin even a thing? (For machines being used as Jenkins agents in particular?)

Reply:
Probably not, but given the range of architectures seen, I was trying to be as compatible as possible.

Reply:
moved to latest (1.16) since there appeared to be more benefits to upgrading rather than staying at 1.14 (bugfixes, apple silicon support)
aa135b481377138f5f449d11c9bc475a60beae25